common/services/user.js

```diff
@@ -8,11 +8,25 @@ const referenceDataService = new ReferenceDataService()
 
 const getAuthHeader = token => ({ Authorization: `Bearer ${token}` })
 
+function retryOnceOn5xx(fun) {
+  return fun().catch(error => {
+    if (error.statusCode < 500) {
```

Comment:
Actually, looking at this again, if the error is malformed and doesn't have the statusCode attribute, this will fail so we will retry. I wonder if we want to change this to have the explicit condition when we retry specified rather then the one when we don't. So something like:
```
if (error.statusCode && error.statusCode >= 500) {
  return fun()
}
  return

Reply:
What about using the [retry-axios](https://www.npmjs.com/package/retry-axios) library?